Src/IronPython/Runtime/Bytes.cs

```diff
@@ -40,6 +40,10 @@ public class Bytes : IList<byte>, IEquatable<Bytes>, ICodeFormattable, IExpressi
             _bytes = bytes.ToArray();
         }
 
+        public Bytes(in ReadOnlySpan<byte> bytes) {
```

Comment:
I think this is what's causing all the test failures. Maybe change it to internal?

Reply:
You are exactly right, it is causing all those test failures. Interestingly, I had no test failures locally, but since then was able to reproduce them, just needed to run `./make clean`. I'm not quite sure I understand why the failures are happening (besides knowing that this constructor is the trigger), perhaps related to/the same as  IronLanguages/ironpython2#347 you mentioned.

Strictly speaking, I don't need this constructor to "Utilize new overloads from .NET 4.6", but looking ahead I thought we will need something like that to handle _bytes-like_, so this was a "trial balloon". Boy, it popped. The good news is that `ReadOnlyMemory` instead of `ReadOnlySpan` works OK.